### PR TITLE
feat(sca): adding registry-url to the cyclonedx output report

### PR DESCRIPTION
**Pull request by itai1357**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**
 
## Description
*adding registry-url to the cyclonedx output report

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/common/output/csv.py [hunk 1]
```diff
@@ -86,10 +91,15 @@ def add_sca_package_resources(self, resource: Record | ExtraResource, git_org: s
                 "Git Repository": git_repository,
                 "Vulnerability": resource.vulnerability_details.get("id"),
                 "Severity": severity,
+                "Description": resource.vulnerability_details.get("description"),
                 "Licenses": resource.vulnerability_details.get("licenses"),
             }
```
Comment (gruebel) [inline checkov/common/output/csv.py:96]: how about moving this dict to a separate variable, ex. `csv_table_entry = {}` then you can check for `CHECKOV_DISPLAY_REGISTRY_URL` and add the `registry_url` to the entry and after that add the entry to the table.

File: checkov/common/sca/output.py [hunk 2]
```diff
@@ -417,24 +427,20 @@ def create_vulnerable_packages_dict(vulnerabilities: list[dict[str, Any]], packa
 
             package_alias = get_package_alias(package["name"], package["version"])
             for cve_idx in package.get('cves_index', []):
-                if package_alias not in vulnerable_packages:
-                    vulnerable_packages[package_alias] = []
-                vulnerable_packages[package_alias].append(vulnerabilities[cve_idx])
+                vulnerable_packages.setdefault(package_alias, []).append(vulnerabilities[cve_idx])
```
Comment (gruebel) [inline checkov/common/sca/output.py:430]: nice 🥇

**Review by gruebel — COMMENTED** [review]
nice, looks good 💪 and please fix the conflict 😉

File: checkov/common/output/csv.py (hunk 1 above)
Comment (itai1357) [reply checkov/common/output/csv.py]: not sure I got it
can you give an example?

**Pull request closed without merging**